python/grass/jupyter/__init__.py

```diff
@@ -13,23 +13,64 @@
 #            License (>=v2). Read the file COPYING that comes with GRASS
 #            for details.
 
-"""Display classes and setup functions for running GRASS GIS in Jupyter Notebooks
+"""The *grass.jupyter* is a convenient GRASS GIS interface for Jupyter notebooks.
+
+Python is a great tool for data science and scientific computing. Jupyter_ is an
+environment with computational notebooks which makes it even better for tool for
+analysis and workflow prototyping. Computational notebooks are documents combining
+code, text, and results such as figures and tables. JupyterLab is an environment where
+you interact with all these parts. You can install it locally on your machine or
+use it online from some service provider.
```

Comment:
Maybe add links to JupyterLab and some service provider?

Reply:
Jupyter is linked in the second sentence of this paragraph (the landing page has JupyterLab as a first item after the banner). Screenshot:

![Screenshot from 2022-04-18 17-41-20](https://user-images.githubusercontent.com/5449060/163882363-bef82206-74b5-43ec-889c-02a63db41813.png)

Binder is an example of service provider linked below. It is not really a typical one, so I'm not sure if it is a good example. Is there a provider we want to advertise? I was thinking a _provider_ might be whoever provides notebook environments at your organization, e.g., the institution itself. Screenshot of the Binder link later:

![Screenshot from 2022-04-18 17-41-42](https://user-images.githubusercontent.com/5449060/163882372-d389eae9-db2b-43be-9b2d-80ad0d6859f9.png)

What do you think?